cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithExternalDBSettingsWithDocker.gradle

```diff
@@ -21,5 +21,8 @@ task prepareDockerNodes(type: net.corda.plugins.Dockerform, dependsOn: 'generate
         cordapps = ["$corda_group:corda-finance-contracts:$corda_release_version",
                     "$corda_group:corda-finance-workflows:$corda_release_version"]
     }
+
     dockerConfig = postgres
+
+    dockerImage = "corda/corda-zulu-java1.8-4.3"
```

Comment:
You could add this value to the tests' `gradle.properties` file:
```
docker_image_name=corda/corda-zulu-java1.8-4.3
```
and then your `.gradle` test files could all refer to it:
```
dockerImage = docker_image_name
```